backends/gdx-backend-lwjgl/pom.xml

```diff
@@ -12,6 +12,13 @@
   <packaging>jar</packaging>
   <name>libGDX LWJGL Backend</name>
 
+  <repositories>
+    <repository>
+      <id>athou-repo</id>
+      <url>https://github.com/Athou/maven-repository/raw/master/repo</url>
```

Comment:
Why don't you use 
```xml
<id>central</id>
    <name>bintray</name>
    <url>http://jcenter.bintray.com</url>
````
?
See https://github.com/DV8FromTheWorld/JAADec for more information :)